Demo/Demo/Card/CardType.swift

```diff
@@ -0,0 +1,25 @@
+enum CardType {
+    case americanExpress, visa, unknown
+
+    var cardNumberIndices: [Int] {
```

Comment:
This variable name is a bit confusing - is this where the space goes between the digits?